marshmallow_dataclass/__init__.py

```diff
@@ -430,8 +460,10 @@ def _get_field_default(field: dataclasses.Field):
     >>> _get_field_default(dataclasses.field())
     <marshmallow.missing>
     """
-    if field.default_factory is not dataclasses.MISSING:
-        return field.default_factory
+    # Remove `type: ignore` when https://github.com/python/typeshed/issues/3246 is fixed
```

Comment:
Maybe this is the issue that should be linked instead:
https://github.com/python/mypy/issues/6910

Reply:
Woops meant to update that. Thanks. Fixed.